packages/engine-server/src/workspace/service.ts

```diff
@@ -1594,41 +1593,41 @@ export class WorkspaceService implements Disposable, IWorkspaceService {
    * Because Dendron workspace relies on major version to be the same, we force a migration if that's not
    * the case
    */
-  async runConfigMigrationIfNecessary({
-    currentVersion,
-    dendronConfig,
-  }: {
-    currentVersion: string;
-    dendronConfig: IntermediateDendronConfig;
-  }) {
-    let changes: MigrationChangeSetStatus[] = [];
-    if (dendronConfig.version !== CURRENT_CONFIG_VERSION) {
-      // NOTE: this migration will create a `migration-config` backup file in the user's home directory
-      changes = await MigrationService.applyMigrationRules({
-        currentVersion,
-        previousVersion: "0.83.0", // to force apply
-        dendronConfig,
-        wsService: this,
-        logger: this.logger,
-        migrations: [CONFIG_MIGRATIONS],
-      });
-      // if changes were made, use updated changes in subsequent configuration
-      if (!_.isEmpty(changes)) {
-        const { data } = _.last(changes)!;
-        dendronConfig = data.dendronConfig;
-      }
-    }
-
-    return changes;
-  }
+  // async runConfigMigrationIfNecessary({
```

Comment:
let's just delete if we don't need it?

Reply:
I left it there to move / re-use it in the still-pending task to handle v4 configs gracefully, but yeah, this will be cleaned up.